AEM/mlperf/README.md

```diff
@@ -164,7 +164,8 @@ bash populate_submission.sh
 ```
 
 ### Validate Submission Checker
-> *For submissions only:* There are several files you should modify before running the submission checker. Here are the changes:
+>**For submissions only:** There are several files you should modify before running the submission checker. Here are the changes:
+
 - In the *systems* folder, there is a JSON file. Change the following fields as needed: *submitter*, *system_name*, and *hw_notes*.
 - In *default.conf*, modify *SYSTEM_DEFAULT* as needed and ensure all paths in *DEL_FILES_DEFAULT* containers your company name rather than "OEM".
```

Comment:
```suggestion
- In *default.conf*, modify *SYSTEM\_DEFAULT* as needed and ensure all paths in *DEL\_FILES\_DEFAULT* containers your company name rather than "OEM".
```